react-events-and-state/src/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: true });
+  }
+
+  render() {
+    return <button onClick={this.handleClick}>
+        {this.state.isClicked ? 'Thanks!' : 'Click Me!'}
```

Comment:
Great use of a ternary! Best way to handle this!

Reply:
Seemed easier than writing out the whole if/then statement, lol.